internal/controller/notebook_controller.go

```diff
@@ -48,6 +47,7 @@ func (r *NotebookReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	}
 
 	if result, err := r.reconcileNotebook(ctx, &notebook); !result.success {
+		log.Info("successfully reconciled notebook")
```

Comment:
This is in a check for: `!result.success`

Reply:
whoops!